cloudbuild/cloudbuild-pr.yaml

```diff
@@ -30,15 +30,6 @@ steps:
   id: Setup connection to GKE
   args: ["container", "clusters", "get-credentials", "$_GKE_CLUSTER", "--project", "$_PROJECT_ID", "--region", "$_LOCATION"]
 
-- name: 'gcr.io/cloud-builders/docker'
```

Comment:
This will not have negative side effects?

Reply:
No.  I was using these for debugging the Cloud Build process.